imap/distributor.go

```diff
@@ -3,12 +3,13 @@ package imap
 import (
 	"bufio"
 	"fmt"
-	"log"
 	"net"
 	"sync"
 
 	"crypto/tls"
 
+	"github.com/go-kit/kit/log"
```

Comment:
Nope. Not really. The aliases are just some sort of placeholder until we can get rid of the stdlib logging. ☺️